src/exporto0o.ts

```diff
@@ -163,9 +163,11 @@ export async function exportToOo(
   }
 
   try {
-    await exec(cmd, { cwd: variables.currentDir, env: { TEXINPUTS: joinEnvPath(`${textemplateDir}`, './') } }); 
+    const { exec } = require('child_process');
+    const path = require('path');
     console.log(`[${plugin.manifest.name}]: export command: ${cmd}`);
-    await exec(cmd, { cwd: variables.currentDir });
+    // orginal `$TEXINPUTS` should be empty, so ignore it.
```

Comment:
You can also add `debugger;console.log('current TEXINPUTS', process.env.TEXINPUTS)` at this line. Then open DevTools to see the value of `TEXINPUTS`.